Makefile

```diff
@@ -81,7 +81,34 @@ fallthrough: submodules
 manifests:
 	@$(WARN) Deprecated. Please run make generate instead.
 
-generate: $(KUSTOMIZE) go.generate manifests.annotate
+generate: $(KUSTOMIZE) go.generate manifests.prepare manifests.annotate
```

Comment:
@jbw976 as discussed:
We'll need `make generate` to be run for helm chart being prepared appropriately. Could you check if this is already called during our release process?

Reply:
May be we should move this process to `make build` just like `values.yaml.tmpl -> values.yaml`.

Reply:
@jbw976 
AFAIU, `make generate` is not called per build. So, we should either:

a. move this process to make build like `values.yaml.tmpl` -> `values.yaml`
or
b. also commit generated files like we do for crd's already, i.e. despite CRD's are generated files, they are also committed.

I couldn't find the proper way to achieve option a, if there is, I would vote for it. Otherwise, option b would also be fine, except we will have duplicate files which will be overridden with `make generate`

Reply:
Ok, I'll go with option b then.

Regarding zombie files, this should not be an issue, please see: https://github.com/crossplaneio/crossplane/pull/1139/files#r373966967